graph_weather/models/atmorep/data/normalizer.py

```diff
@@ -0,0 +1,78 @@
+import numpy as np
+
+
+class FieldNormalizer:
+    def __init__(self, config, stats_dir, create_stats=False):
+        """
+        Args:
+            config (AtmoRepConfig): Configuration containing input_fields.
+            stats_dir (Path): Directory containing per-field statistics files.
+            create_stats (bool): If True, calculate statistics; otherwise, load from disk.
+        """
+        self.config = config
+        self.stats_dir = stats_dir
+        self.stats = {}
+        if create_stats:
+            self.calculate_stats()
+        else:
+            for field in config.input_fields:
+                stats_file = stats_dir / f"{field}_stats.npy"
+                if stats_file.exists():
+                    loaded = np.load(stats_file, allow_pickle=True).item()
+                    self.stats[field] = loaded
+                else:
+                    raise Exception(f"Stats file for {field} not found")
+
+    def calculate_stats(self):
+        """
+        Placeholder implementation to calculate statistics.
+        In practice, you would compute the mean and std from your dataset.
+
+        Returns:
+            dict: A dictionary of statistics for each field.
+        """
+        self.stats = {field: {"mean": 0.0, "std": 1.0} for field in self.config.input_fields}
```

Comment:
```suggestion
        return NotImplementedError
```
If this isn't actually computing the mean/stddev then it should return a NotImplementedError, not some fake stats, as that can cause downstream errors.